### PR TITLE
Minimal tags added to card

### PR DESCRIPTION
**Pull request by PukiPuki**

### CODE REVIEW

File: src/main/java/seedu/address/storage/XmlAdaptedCard.java [hunk 1]
```diff
@@ -62,7 +62,8 @@ public Card toModelType() throws IllegalValueException {
         if (!Card.isValidCard(this.back)) {
             throw new IllegalValueException(Card.MESSAGE_CARD_CONSTRAINTS);
         }
-        return new Card(front, back);
+        // change this later
+        return new Card(front, back, null);
```
Comment (yong-jie) [inline src/main/java/seedu/address/storage/XmlAdaptedCard.java]: Is this supposed to be changed before the PR?

File: src/main/java/seedu/address/model/card/Card.java [hunk 2]
```diff
@@ -27,10 +31,14 @@
     private final String front;
     private final String back;
 
-    public Card(String front, String back) {
-        requireAllNonNull(front, back);
+    private final UniqueTagList tags;
+
+    public Card(String front, String back, Set<Tag> tags) {
+        requireAllNonNull(front, back, tags);
```
Comment (jethrokuan) [inline src/main/java/seedu/address/model/card/Card.java]: You require that tags be non-null, yet later on you specifically pass `null` to the component.

File: src/test/java/seedu/address/logic/commands/AddCardCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.address.model.card.Card;
+import seedu.address.testutil.CardBuilder;
+
+public class AddCardCommandTest {
```
Comment (jethrokuan) [inline src/test/java/seedu/address/logic/commands/AddCardCommandTest.java]: Please add a test for cards without tags.

**Comment by yong-jie** [comment]
Resolves #22, #23

**Comment by jethrokuan** [comment]
Deprecated in favour of #56

**Pull request closed without merging**